src/Microsoft.AspNetCore.Blazor.Build/targets/Publish.targets

```diff
@@ -0,0 +1,59 @@
+﻿<Project>
+  <PropertyGroup>
+    <BlazorLinkOnBuild Condition="'$(Configuration)'=='Release' AND '$(BlazorLinkOnBuild)'==''">true</BlazorLinkOnBuild>
+    <BlazorPublishDistDir>$(AssemblyName)\dist\</BlazorPublishDistDir>
+
+    <!-- Disable unwanted parts of the default publish process -->
+    <CopyBuildOutputToPublishDirectory>false</CopyBuildOutputToPublishDirectory>
+    <CopyOutputSymbolsToPublishDirectory>false</CopyOutputSymbolsToPublishDirectory>
+    <PreserveCompilationContext>false</PreserveCompilationContext>
+    <RazorCompileOnPublish>false</RazorCompileOnPublish>
```

Comment:
I think we already turn `RazorCompileOnPublish` off in the Razor targets. The `RazorCompileOnBuild`/`RazorCompileOnPublish` properties refer to the way that MVC uses Razor and are thus never relevant for Blazor.

Reply:
It looks like we turn `RazorCompileOnPublish` **on** in the Razor targets: https://github.com/aspnet/Razor/blob/dev/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets#L94

Also in practice there's a build error if we don't explicitly set it to `false` here (*The type or namespace name 'Razor' does not exist in the namespace 'Microsoft.AspNetCore' (are you missing an assembly reference?)*).